.github/workflows/test-runner-on-perf.yml

```diff
@@ -25,6 +25,9 @@ jobs:
           LATEST_VERSION=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
           echo "Latest Version: $LATEST_VERSION"
           echo "::set-output name=LATEST_VERSION::$LATEST_VERSION"
+          LATEST_GITHASH=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.gitHash')
```

Comment:
I'm sure there's a way to capture and assign multiple results in a single jq but I'm not sure how to do it. Probably not worth the effort. ... well, I had to find out, so here it is. Not sure it's worth doing but it's good to know it's possible.

```
read -r LATEST_GITHASH LATEST_VERSION <<<$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.gitHash,.semVer|rtrimstr("-SNAPSHOT")')
echo "Latest git hash: $LATEST_GITHASH"
echo "Latest Version: $LATEST_VERSION"
```

Reply:
That's a great idea - thanks!! This is great b/c this will keep the latest version and githash in sync since we're grabbing it from the same request. 
I tested it w/ a local script, but for some reason it failed in the action. I'm a bit stumped: https://github.com/DataBiosphere/jade-data-repo/runs/2409744876?check_suite_focus=true#step:6:21

Reply:
Try `read -d "\n" ...`, which tells `read` to use CR as a delimiter instead of space. The different behavior could be due to different versions of bash.

Reply:
Ok, I've tried a bunch of different variations. I have no idea why it's been wonky. I'm inclined to just revert - now not worth time investment.